docs/docusaurus/sidebars.js

```diff
@@ -483,8 +483,6 @@ module.exports = {
             },
             'terms/expectation_suite',
             'terms/metric',
-            'conceptual_guides/metricproviders',
```

Comment:
is `conceptual_guides/metricproviders` also duplicated?

Reply:
@HaebichanGX yes it was. It has been corrected.

Reply:
ah I see